core/domain/exp_domain_test.py

```diff
@@ -1883,6 +1884,20 @@ def test_validation_fails_with_invalid_contributor_id_in_contributor_ids(
             'Expected each id in contributor_ids to be string, received 2'):
             self.exp_summary.validate()
 
+    def test_is_private(self):
+        self.assertTrue(self.exp_summary.is_private())
+        self.exp_summary.status = constants.ACTIVITY_STATUS_PUBLIC
+        self.assertFalse(self.exp_summary.is_private())
+
+    def test_is_solely_owned_by_user(self):
```

Comment:
Ditto, ensure tests are as comprehensive as possible and cover all the possible situations.